sources/packages/backend/apps/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -118,6 +124,8 @@ export class EducationProgramOfferingAPIOutDTO {
   lacksStudyBreaks: boolean;
   offeringIntensity: OfferingIntensity;
   yearOfStudy: number;
+  isAviationOffering: AviationYesNoOptions;
```

Comment:
We have a generic `OfferingYesNoOptions` and is it required to create separate one for aviation ?